tests/TestCase/Validation/ValidationTest.php

```diff
@@ -3103,4 +3103,18 @@ public function testImageWidth()
         $this->assertFalse(Validation::imageWidth($upload, '>', 2000));
         $this->assertFalse(Validation::imageWidth($upload, '==', 3000));
     }
+
+    /**
+     * Test color
+     */
+    public function testColor()
+    {
+        $this->assertTrue(Validation::color('#F01234'));
+        $this->assertTrue(Validation::color('#F56789'));
+        $this->assertTrue(Validation::color('#abcdef'));
+        $this->assertTrue(Validation::color('#ABCDEF'));
+
+        $this->assertFalse(Validation::color('#fff'));
+        $this->assertFalse(Validation::color('ffffff'));
```

Comment:
At the moment, `input[type=color]` accepts a simple color only. 
https://developers.whatwg.org/states-of-the-type-attribute.html#color-state-(type=color)
But since Validator class doesn't allow people to change the signature without breaking BC, should I now add an additional field to the validation method for future extension? Or should I name the method `simpleColor`?